application/src/main/java/eu/europa/ec/fisheries/uvms/activity/message/producer/ConfigMessageProducerBean.java

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
This one I am starting to worry about a little bit if it should be removed, although it isn't used as far as I can tell...

Reply:
I think it's cool. It can send messages but if no-one calls it it will not send any messages. And we have an ActivityConfigProducerBean that is almost identical, and which is actually called.